schemas/entailment.py

```diff
@@ -0,0 +1,14 @@
+"""
+Textual Entailment Schema
+"""
+import datasets
+
+
+features = datasets.Features(
+    {
+        "id": datasets.Value("string"),
+        "premise": datasets.Value("string"),
+        "hypothesis": datasets.Value("string"),
+        "label": datasets.Value("string"),
```

Comment:
add even more guide rails and make this 
`"label": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"])`
or too strict?

Reply:
Yeah, I'm torn. Let's leave this unrestricted for now and see if it becomes a problem and we can flag this for fixing in a post-hackathon cleanup pass.